core/bin/external_node/src/config/mod.rs

```diff
@@ -136,6 +136,14 @@ pub struct OptionalENConfig {
     latest_values_cache_size_mb: usize,
     /// Enabled JSON RPC API namespaces.
     api_namespaces: Option<Vec<Namespace>>,
+    /// Whether to support methods installing filters and querying filter changes.
+    ///
+    /// Rationale for setting this to `false`:
```

Comment:
Bikeshedding: I readily admit I'm not an expert on English, but the first time I read this comment, it looked like it explained why `false` is the default value for this field (which isn't the case; the default value is `true`). IMO, it'd be more straightforward to formulate this something like "When to set this value to `false`".